test/__tests__/uppy-server.js

```diff
@@ -0,0 +1,65 @@
+/*global jest:false, test:false, expect:false, describe:false*/
```

Comment:
adding `jest: true` to eslintrc didn't seem to work with the `test` global. I also thought since the entire codebase does not consist of tests alone, it might be better to not deceive eslint with the assumption.